catalyst-gateway/bin/src/settings/signed_doc.rs

```diff
@@ -22,10 +19,9 @@ pub(crate) struct EnvVars {
     /// The Catalyst Signed Document threshold, document cannot be too far behind.
     past_threshold: Duration,
 
-    /// The Catalyst Signed Document Admin keys map from the `SIGNED_DOC_ADMIN_KEYS` env
-    /// var. Each Admin key entry is a pair of `CatalystId` string and hex encoded
-    /// `VerifyingKey` separated by ';' character.
-    admin_keys: HashMap<catalyst_signed_doc::CatalystId, ed25519_dalek::VerifyingKey>,
+    /// The Catalyst Signed Document Admin Catalyst ID from the `SIGNED_DOC_ADMIN_KEYS`
+    /// env.
+    admin_key: catalyst_signed_doc::CatalystId,
```

Comment:
Lets make it optional so we would not need to panic and will not break our original flow of the `cat-gateway`.
Instead of panicking lets emit ERROR log, and that's it.